plugins/modules/lke_cluster_info.py

```diff
@@ -143,14 +143,34 @@ def _get_cluster_from_kwargs(self, **kwargs: Any) -> LKECluster:
 
         return self.fail(msg="one of `label` or `id` must be specified")
 
+    def _safe_get_cluster_acl(
```

Comment:
Given this is used in both `lke_cluster` and `lke_cluster_info`, I'm thinking it might make sense to move it into the shared helper package 👍